BDArmory/Competition/BDATournament.cs

```diff
@@ -2126,14 +2126,13 @@ bool GenerateCleanGame()
                     kerbal.isBadass = true; // Make them badass.
                     KerbalRoster.SetExperienceTrait(kerbal, KerbalRoster.pilotTrait); // Make the kerbal a pilot (so they can use SAS properly).
                     KerbalRoster.SetExperienceLevel(kerbal, KerbalRoster.GetExperienceMaxLevel()); // Make them experienced.
-                    kerbal.courage = 0.5f;
```

Comment:
These changes (also in some of the other PRs) are undoing changes that are intended to be in dev (possibly included from Josue's PR?), e.g., `kerbal.courage = 0.5f;` and `sine`->`since` were added in commits 5723d7fa1 and 8385446e3, respectively.
Could you go over these PRs again when you have time and filter out the changes that shouldn't be here?

Reply:
Done.